main.js

```diff
@@ -0,0 +1,44 @@
+"use strict";
+
+var Item = function(attribute) {
+  this.title = attribute.title;
+  this.complete = false;
+};
+
+var ToDoList = function() {
+  this.itemArr = [];
+};
+
+var myList = new ToDoList();
+
+
+ToDoList.prototype.addItem = function(item) {
+  this.itemArr.push(item);
+};
+
+Item.prototype.completeItem = function() {
+  this.complete = true;
+  $('ul').empty();
+  myList.renderToDom();
```

Comment:
It seems a bit weird to have these two lines in the `completeItem` method. The method needs to be aware of `myList` for it to work. Instead I would move these to the click event handler on line 34.